src/pages/billing/Project.js

```diff
@@ -221,10 +224,42 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
     _.map('workspace', workspaces)
   ), [billingProject, workspaces])
 
+  const spendReportKey = 'spend report'
+  const maybeLoadTotalCost = reportErrorAndRethrow('Unable to retrieve spend report data')(async () => {
+    if (!updatingTotalCost && totalCost === null && tab === spendReportKey) {
+      setUpdatingTotalCost(true)
+      const endDate = new Date().toISOString().slice(0, 10)
+      const startDate = new Date((Date.now() - spendReportLengthInDays * 24 * 60 * 60 * 1000)).toISOString().slice(0, 10)
+      const spend = await Ajax(signal).Billing.getSpendReport({ billingProjectName: billingProject.projectName, startDate, endDate })
+      const costFormatter = new Intl.NumberFormat(navigator.language, { style: 'currency', currency: spend.spendSummary.currency })
+      setTotalCost(costFormatter.format(spend.spendSummary.cost))
+      setUpdatingTotalCost(false)
+    }
+  })
+  maybeLoadTotalCost()
+
+  const CostCard = ({ title, amount, ...props }) => {
+    return div({
+      ...props,
+      style: {
+        ...Style.elements.card.container,
+        backgroundColor: 'white',
+        width: 250,
+        padding: undefined,
+        boxShadow: undefined,
+        gridRowStart: 2
+      }
+    }, [
+      div({ style: { flex: 'none', padding: '10px 20px' } }, [
```

Comment:
I am using pixels since these are just padding values and don't necessarily need to grow. I am trying to approximate UX mockups with respect to padding and font size.

Reply:
Or shrink? '0.625rem 1.25rem' would get you the same initial values.

Reply:
Good point. I changed all the pixels to rems.